### PR TITLE
Updates to handle untrapped exceptions formerly mapped to '1' but w/o traceback.

### PR DESCRIPTION
**Pull request by jaytmiller**
Didn't run this on Batch but I think the doctests either didn't change or illustrate what this does.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -24,6 +24,7 @@ install_requires =
     awscli
     spec-plots==1.34.6
     psutil
+    voila<0.2
```
Comment (bhayden53) [inline setup.cfg:27]: what's this for?

Comment (jaytmiller) [reply setup.cfg:27]: Good catch.   I misread a pip error which really shouldn't have been happening either.   (Pip reported broken dependencies from some past install which did not report them...  while I was installing caldp.  I thought it was odd.)

Comment (jaytmiller) [reply setup.cfg]: Should be gone now.

**Comment by bhayden53** [comment]
LGTM, don't care about the 0.32% codecov complaint

**Pull request merged**